### PR TITLE
Removes controversy

### PR DESCRIPTION
**Pull request by Zth--**
Woop woop

### CODE REVIEW

File: code/game/gamemodes/endgame/supermatter_cascade/blob.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#define BLUESPACELEAK_FLAT
+//#define BLUESPACELEAK_FLAT if you touch this I'll cut your fingers
```
Comment (ghost) [inline code/game/gamemodes/endgame/supermatter_cascade/blob.dm:1]: Touched it

Comment (clusterfack) [inline code/game/gamemodes/endgame/supermatter_cascade/blob.dm]: 🔪 🖐

**Comment by Aurx** [comment]
I don't like this change.  
Please tag it as [Controversial]

**Comment by mrpain666** [comment]
damn I wish I had a tag named after me

**Comment by Intigracy** [comment]
fyi since this PR is a meme and nondescriptive

current supermatter cascade
d096fc6c-e688-11e6-9e40-26478700808e

with change (ignore edges on first picture, fixed in second)
1779fc2a-e65b-11e6-83cb-50454f44215b

2d797ae2-e65f-11e6-9eab-c699d88b3962

both were added in https://github.com/d3athrow/vgstation13/pull/13715 but the first one is the one enabled

**Comment by Aurx** [comment]
Should honestly be a config option not a define, if it's going to have multiple settings.

**Comment by Riekland** [comment]
Literally going against the will of the majority, I'm going to start a gommunist revolution if this shit goes through

**Comment by Zth--** [comment]
Oh my god what have I done

**Comment by gbasood** [comment]
@Riekland 
oh no we're gonna have a revolt guys!!!
fuck I hate meme PR titles

**Comment by Riekland** [comment]
This is literally unacceptable, writing a perma request as we speak

**Pull request merged**